docs/services/data/deploy-model.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Deploy a Model"
+linkTitle: "Deploy Model"
+weight: 60
+type: "docs"
+tags: ["data management", "ml", "model training"]
+description: "Deploy an image classification model to a robot."
+# SME: Aaron Casas
+---
+
+On the [**DATA** tab](https://app.viam.com/data/view) in the Viam app, navigate to the **MODELS** sub-tab.
+
+![The trained model](../img/stars-model.png)
+
+Click the **COPY** button to copy the model information.
+The model information looks like this:
+
+```json
+{
+    "package": "a1b23cde-fgh4-56i-j78k-l90123m45n6o/stars-model",
+    "version": "1234567891011",
+    "name": "stars-model"
+}
+```
+
+Next, navigate to your robot's [**CONFIG** tab](https://app.viam.com/robot) and click on the **SERVICES** subtab.
+
+Scroll to the bottom and create a new service with the **Type** `ml_models`.
+
+![Create a machine learning models service](../img/ml-models-service.png)
+
+The created ML Models panel lists all available models.
+
+![Create a machine learning models service](../img/available-models.png)
+
+To add a model to your robot, select it and click on the move right button.
+
+![Create a machine learning models service](../img/added-model.png)
+
+Save the configuration and your model will be added to your robot at <file>/root/.viam/packages/<model-name></file>.
+
+To make use of your new model, use the Vision Service.
+Still on the **SERVICES** subtab add a new service with **Type** `Vision` and give it a name.
+
+Create a [tflite_classifier` classifier](../../vision/#classification) by adding the following configuration in the new Vision Service panel.
+
+```json
+{
+  "register_models": [
+    {
+      "type": "tflite_classifier",
+      "parameters": {
+        "num_threads": 1,
+        "model_path": "${packages.<model-name>}/<model-name>.tflite",
+        "label_path": "${packages.<model-name>}/labels.txt"
+      },
+      "name": "<classifier-name>"
+    }
+  ]
+}
+```
+
+Save the configuration.
+
+## Add a camera component and a "transform" model
+
+You cannot interact directly with the [Vision Service](/services/vision/).
+To be able to interact with the Vision Service you must:
+
+1. Configure a physical [camera component](../../../components/camera) to wrap the service.
+2. Configure a [transform camera](../../../components/camera/transform) to view output from the detector overlaid on images from the physical camera.
```

Comment:
Optional, but might be useful to include an example config, since the classifier name must match `classifier-name` above.

e.g. copied over from the QA plan:
Add a camera Transform by creating a new component of Type camera and Model transform. In the config, paste the following:
```
{
 "pipeline": [
   {
     "type": "classifications",
     "attributes": {
       "confidence_threshold": 0.5,
       "classifier_name": "my_classifier"
     }
   }
 ],
 "source": "cam"
}
```